client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -190,5 +190,25 @@ describe('PythonCodePreprocessorService', function() {
         }).toThrow(new Error('Incomplete line: missing "(" in def statement.'));
       }
     );
-  })
+
+    it('should add correct performance test information to skeleton code',
+      function() {
+        var performanceTests = [
+          {
+            "input_data_atom": "na ",
+            "transformation_function": "extendString",
+            "expected_performance": "linear",
+            "evaluation_function": "katamariDamashi"
+          }
+        ];
+
+        outputCode = PythonCodePreprocessorService.generatePerformanceTestCode(
+                performanceTests);
+
+        expect(outputCode).toContain('katamariDamashi');
```

Comment:
It seems like, long-term, that's going to cause us problems, though, because as we change the performance code we're going to have to update all this hardcoded output code to be consistent with whatever paradigm we choose to enforce going forward. Part of the reason I was only testing the contains was that I didn't see any similar method for correctness, and I'd rather just check for what needs to be in the output code rather than check to ensure that the arbitrary structure is identical.

I'm not suggesting that we NOT do the code equality check, but I think we should be mindful of accruing technical debt. Perhaps it'd be worth writing a helper method to autogenerate the code, test to make sure THAT works, and then check the autogenerated code to make sure it contains the correct parameters for correctness, performance, style, and buggy_output tests.